packages/fiori/src/MediaGallery.ts

```diff
@@ -260,12 +78,189 @@ const metadata = {
+	 * @name sap.ui.webc.main.MediaGallery.prototype.layout
+	 * @defaultvalue "Auto"
+	 * @public
+	 */
+	@property({ type: MediaGalleryLayout, defaultValue: "Auto" })
+	layout!: MediaGalleryLayout;
+
+	/**
+	 * Determines the horizontal alignment of the thumbnails menu
+	 * vs. the central display area.
+	 * <br><br>
+	 * Available options are:
+	 * <ul>
+	 * <li><code>Left</code></li>
+	 * <li><code>Right</code></li>
+	 * </ul>
+	 *
+	 * @type {sap.ui.webc.fiori.types.MediaGalleryMenuHorizontalAlign}
+	 * @name sap.ui.webc.main.MediaGallery.prototype.menuHorizontalAlign
+	 * @defaultvalue "Left"
+	 * @public
+	 */
+	@property({ type: MediaGalleryMenuHorizontalAlign, defaultValue: MediaGalleryMenuHorizontalAlign.Left })
+	menuHorizontalAlign!: MediaGalleryMenuHorizontalAlign;
+
+	/**
+	 * Determines the vertical alignment of the thumbnails menu
+	 * vs. the central display area.
+	 * <br><br>
+	 * Available options are:
+	 * <ul>
+	 * <li><code>Top</code></li>
+	 * <li><code>Bottom</code></li>
+	 * </ul>
+	 *
+	 * @type {sap.ui.webc.fiori.types.MediaGalleryMenuVerticalAlign}
+	 * @name sap.ui.webc.main.MediaGallery.prototype.menuVerticalAlign
+	 * @defaultvalue "Bottom"
+	 * @public
+	 */
+	@property({ type: MediaGalleryMenuVerticalAlign, defaultValue: MediaGalleryMenuVerticalAlign.Bottom })
+	menuVerticalAlign!: MediaGalleryMenuVerticalAlign;
+
+	/**
+	 * Determines the actual applied layout type
+	 * (esp. needed when the app did not specify a fixed layout type
+	 * but selected <code>Auto</code> layout type).
+	 * <br><br>
+	 * Possible values are:
+	 * <ul>
+	 * <li><code>Vertical</code></li>
+	 * <li><code>Horizontal</code></li>
+	 * </ul>
+	 *
+	 * @type {sap.ui.webc.fiori.types.MediaGalleryLayout}
+	 * @defaultvalue "Vertical"
+	 * @private
+	 */
+	@property({ type: MediaGalleryLayout, defaultValue: "Vertical" })
+	effectiveLayout!: MediaGalleryLayout;
+
+	/**
+	 * Defines the current media query size.
+	 *
+	 * @private
+	 */
+	@property()
+	mediaRange!: string;
+
+	/**
+	 * The number of items in the overflow.
+	 *
+	 * @private
+	 */
+	@property({ validator: Integer, noAttribute: true, defaultValue: 0 })
+	_overflowSize!: number;
+
+	/**
+	 * Defines the component items.
+	 *
+	 * <br><br>
+	 * <b>Note:</b> Only one selected item is allowed.
+	 *
+	 * <br><br>
+	 * <b>Note:</b> Use the <code>ui5-media-gallery-item</code> component to define the desired items.
+	 *
+	 * @type {sap.ui.webc.fiori.IMediaGalleryItem[]}
+	 * @name sap.ui.webc.main.MediaGallery.prototype.default
+	 * @slot items
+	 * @public
+	 */
+	@slot({
+		type: HTMLElement, individualSlots: true, invalidateOnChildChange: true, "default": true,
+	})
+	items!: Array<MediaGalleryItem>;
+
+	_itemNavigation: ItemNavigation;
+	_onResize: () => void;
+	_selectedItem?: MediaGalleryItem;
+
 	constructor() {
 		super();
 		this._onResize = this._updateLayout.bind(this);
-		this._selectedItem = null;
 		this._initItemNavigation();
```

Comment:
You already declared _selectedItem as undefined and you already moved initItemNavigation content out of it. These two seems to be redudant: 
		this._initItemNavigation();
		this._selectedItem = undefined;